src/components/piechart/PieChart.stories.mdx

```diff
@@ -8,24 +8,27 @@ import { SVGIcon } from "../svgicon";
 
 # PieChart
 
-## Props
+## PieChart properties
 
-- `size`: `number` — **required**
-- `progress`: `number` — **required**
-- `strokeWidth`: `number` — **required**
-- `color`: `danger` | `warning` | `success` | `notification`;
+|  properties | required | type                                                                                       | description              |
+| ----------: | :------: | :----------------------------------------------------------------------------------------- | :----------------------- |
+|        size |   true   | number                                                                                     | width and height in `px` |
+|    progress |   true   | number                                                                                     |                          |
+| strokeWidth |   true   | number                                                                                     |                          |
```

Comment:
I don't know what "strokeWidth" means so maybe add description to this one? :)

Reply:
Fixed

<img width="1051" alt="Zrzut ekranu 2021-04-19 o 12 27 17" src="https://user-images.githubusercontent.com/58164749/115222078-a1216f80-a10a-11eb-8694-4c808226b97d.png">